rotkehlchen/chain/evm/decoding/curve/curve_cache.py

```diff
@@ -265,6 +265,9 @@ def _ensure_curve_tokens_existence(
                         token_kind=EvmTokenKind.ERC20,
                         weight=ONE,
                     )],
+                    decimals=18,  # 18 is the default for old gauges that don't implement the decimals method https://t.me/curvefi/654915  # noqa: E501
```

Comment:
```suggestion
                    decimals=18,  # all gauges have 18 decimals https://t.me/curvefi/654915. Hard code since some contracts have no name/symbol/decimals.  # noqa: E501
```